cognite/seismic/protos/query_service_messages.proto

```diff
@@ -267,6 +267,8 @@ message GetFileResponse {
     string path = 3;
     string survey_name = 4;
     string last_step = 5;
+    int32 inline_offset = 6;
```

Comment:
These should perhaps be Int32Value messages, so their presence or absence can be clearly visible. Having these default to 0 is not a great experience.
Alternatively the protobuf definition itself should include default values.